test/performance/README.md

```diff
@@ -17,6 +18,55 @@ We can use the `perftool` to do the performance test for the multicluster contro
 By default, this performance test will create 1k clusters and create 5 manifestworks on the each clsuter, you can
 configure the `--count` and `--work-count` in the `hack/performance.sh` to modify the cluster and manifestwork conuts.
 
+By default, we use the below manifestwork as the workload
+
+```yaml
+apiVersion: work.open-cluster-management.io/v1
+kind: ManifestWork
+metadata:
+  name: <cluster-name>-work-<work-number>
+spec:
+  workload:
+    manifests:
+    - apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        name:   <cluster-name>-work-<work-number>
+        namespace: default
+      data:
+        test: "I'm a test configmap"
+```
+
+You can customize your own workload by following steps
+
+1. Define your manifestworks in a directory and use `` annotation to specify your expected manifestwork counts, e.g.
+
+```yaml
+apiVersion: work.open-cluster-management.io/v1
+kind: ManifestWork
+metadata:
+  name: work-1
+  annotations:
+    "perftest.open-cluster-management.io/expected-work-count": "10"
+spec:	
+  workload:
+    manifests: <your-manifests>
+
```

Comment:
you may need `---` to separate the yaml file

Reply:
added